app/main/logging_config.py

```diff
@@ -0,0 +1,25 @@
+''' Logging info, debug with coloredlogs '''
```

Comment:
I've a doubt? How are we importing this in Flask and using the settings there?

Can we use this inside other files? If yes, how?

Reply:
importing the file just as module should do right ?

Reply:
Please setup a small logging instance, like in manage.py with run command log ,"starting server on port xxxx"